bin/update_release_draft

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env ruby
```

Comment:
nit: maybe worth naming this file with dashes for consistency with other files in bin/

Reply:
Thanks! Fixed!